css/light-styles.css

```diff
@@ -45,6 +45,21 @@ table th, table td {
   color:#036;
 }
 
+.subject-select-container
+{
+   border-color: #c69;
+}
+
+.subjectselect
+{
+   color: #c69;
+   font-size: 1.35em;
+   background-color: transparent;
+   background-image: url("../img/Select.png");
+   background-position: 87% center;
+   background-repeat: no-repeat;
+   -webkit-appearance: button;  
+
```

Comment:
-webkit-appearance works only for a few browsers. See https://developer.mozilla.org/en/docs/Web/CSS/-moz-appearance